jpa/hibernate4_3/src/test/java/org/jboss/as/jpa/hibernate4/scan/ScannerTest.java

```diff
@@ -159,14 +147,6 @@ protected void addPackageToClasspath(File... files) throws MalformedURLException
         Thread.currentThread().setContextClassLoader(classLoader);
     }
 
-    protected void addPackageToClasspath(URL... urls) throws MalformedURLException {
```

Comment:
Why has so much hibernate code and tests been removed? There does not seem to be any link to a JIRA explaining the reasoning.

Reply:
unused code, no other reason.